.gitmodules

```diff
@@ -0,0 +1,5 @@
+[submodule "circomlib"]
+	path = packages/circuits/circuits/circomlib
+	url = https://github.com/iden3/circomlib.git
+        shallow = true
+        active = true
```

Comment:
can we set to branch `v2.0.5`?

Reply:
unfortunately no, only branches can be set, not tags/commits. But this won't break already released packages. It will just affect our development environments. I think this is okay because we'll get new updates automatically.

If it does break things in the future we can fork the circomlib repo and maintain our own branch. We could also fork circomlib now into the unirep namespace and point to that

Reply:
yeah we can maintain the circomlib update in our repo. (I think we don't have to for it)